src/target/riscv/riscv.h

```diff
@@ -129,6 +129,22 @@ struct reg_name_table {
 	char **reg_names;
 };
 
+typedef struct riscv_mem_access_info {
+	target_addr_t target_address;
+
+	union {
+		uint8_t *read;
+		const uint8_t *write;
+	} buffer;
+
+	uint32_t element_size;
```

Comment:
Could this be called just `size` for simplicity?

The name `size` is well-established in memory read/write functions across the whole OpenOCD, so I expect no confusion here.

Reply:
Addressed